src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -1172,6 +1245,7 @@ class CreateVmModal extends React.Component {
                 userPassword: this.state.userPassword,
                 rootPassword: this.state.rootPassword,
                 userLogin: this.state.userLogin,
+                sshKeys: this.state.sshKeys.map(key => key.value),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1150-20231003-232224-a08f566d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)